.github/workflow/nodejs.yml

```diff
@@ -0,0 +1,27 @@
+name: Node-CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 13.x]
+        os: [ubuntu-latest, macOS-latest]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: npm install, build and test
+        run: |
+          npm ci
+          npm run build --if-present
+          npm test
```

Comment:
There is no `test` or `ci` commands in this project – https://github.com/nfarina/homebridge/blob/master/package.json#L6

Reply:
This is just like [hap-nodejs](https://github.com/KhaosT/HAP-NodeJS/raw/master/.github/workflows/nodejs.yml)

Reply:
`hap-nodejs` has more elaborate scripts section https://github.com/KhaosT/HAP-NodeJS/blob/master/package.json#L31-L35

You can try to run `npm test` locally and see that it would error out.

This projects needs tests before it could be automated to run tests in CI. :)

Reply:
Do you wanna change it?

Reply:
I mean, it will fail CI, because test script isn't defined. If you add some rudimentary test script, it would a good start and it will make that CI runs meaningful, otherwise, all we'd get is failed status marks on each PR, not really useful.